app/[locale]/academics/departments/[name]/page.tsx

```diff
@@ -1,9 +1,239 @@
-import WorkInProgress from '~/components/work-in-progress';
+import Image from 'next/image';
+import Link from 'next/link';
+import { ImLab } from 'react-icons/im';
+import { MdBadge, MdMilitaryTech } from 'react-icons/md';
 
-export default function Department({
-  params: { locale },
+import { getTranslations } from '~/i18n/translations';
+import HorsesRunning from '~/components/horses-running';
+import DepartmentGallery from '~/components/department-gallery';
+import Subnav from '~/components/subnav';
+
+export default async function Department({
+  params: { locale, name },
 }: {
   params: { locale: string; name: string };
 }) {
-  return <WorkInProgress locale={locale} />;
+  const text = (await getTranslations(locale)).Departments;
+
+  const departmentData = {
+    name: name.toUpperCase(),
+    titleImage:
+      'https://www.figma.com/file/2UXH7QR1dUT8KXgdGuYaC1/image/cf46d1812e6285efda1f8172b7fc3634c6d76130',
+  };
+  const hod = {
+    name: 'John does',
+    image:
+      'https://www.figma.com/file/2UXH7QR1dUT8KXgdGuYaC1/image/cf46d1812e6285efda1f8172b7fc3634c6d76130',
+    message:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa ut adipisci error voluptates reiciendis reprehenderit veniam consequatur vitae a atque?',
+    contacts: {
+      email: 'abc@gmail.com',
+      phone: ['1234567890', '1234567890'],
+    },
+  };
+
+  return (
+    <main className=" w-screen">
+      <header
+        style={{
+          backgroundImage: `url(${departmentData.titleImage})`,
+          backgroundSize: 'cover',
+          backgroundPosition: 'center',
+        }}
+      >
```

Comment:
Why do we have inline styles over Tailwind? Please fix. We can set bg images using Tailwind as well, see <https://tailwindcss.com/docs/background-image#arbitrary-values>

Reply:
Thank you for informing me. I wasn't aware of this, but now that I know, I'll make the necessary changes